disperser/cmd/controller/flags/flags.go

```diff
@@ -0,0 +1,182 @@
+package flags
+
+import (
+	"time"
+
+	"github.com/Layr-Labs/eigenda/common"
+	"github.com/Layr-Labs/eigenda/common/aws"
+	"github.com/Layr-Labs/eigenda/common/geth"
+	"github.com/Layr-Labs/eigenda/core/thegraph"
+	"github.com/Layr-Labs/eigenda/indexer"
+	"github.com/urfave/cli"
+)
+
+const (
+	FlagPrefix   = "controller"
+	envVarPrefix = "CONTROLLER"
+)
+
+var (
+	DynamoDBTableNameFlag = cli.StringFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "dynamodb-table-name"),
+		Usage:    "Name of the dynamodb table to store blob metadata",
+		Required: true,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "DYNAMODB_TABLE_NAME"),
+	}
+	BlsOperatorStateRetrieverFlag = cli.StringFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "bls-operator-state-retriever"),
+		Usage:    "Address of the BLS Operator State Retriever",
+		Required: true,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "BLS_OPERATOR_STATE_RETRIVER"),
+	}
+	EigenDAServiceManagerFlag = cli.StringFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "eigenda-service-manager"),
+		Usage:    "Address of the EigenDA Service Manager",
+		Required: true,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "EIGENDA_SERVICE_MANAGER"),
+	}
+	UseGraphFlag = cli.BoolFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "use-graph"),
+		Usage:    "Whether to use the graph node",
+		Required: true,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "USE_GRAPH"),
+	}
+	IndexerDataDirFlag = cli.StringFlag{
+		Name:   common.PrefixFlag(FlagPrefix, "indexer-data-dir"),
+		Usage:  "the data directory for the indexer",
+		EnvVar: common.PrefixEnvVar(envVarPrefix, "INDEXER_DATA_DIR"),
+		Value:  "./data/",
+	}
+	// EncodingManager Flags
+	EncodingPullIntervalFlag = cli.DurationFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "encoding-pull-interval"),
+		Usage:    "Interval at which to pull from the queue",
+		Required: true,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "ENCODING_PULL_INTERVAL"),
+	}
+	AvailableRelaysFlag = cli.IntSliceFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "available-relays"),
+		Usage:    "List of available relays",
+		Required: true,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "AVAILABLE_RELAYS"),
```

Comment:
How does this work? Each relay has a integer assigned to it?

Reply:
Yes, this will be a list of integer representing the disperser's relay keys